enrolments/tests/views/test_session.py

```diff
@@ -0,0 +1,80 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from accounts.models import User
+from enrolments.models import Session
+from enrolments.serializers import SessionSerializer
+
+
+class SessionTestCase(APITestCase):
+    def setUp(self):
+        self.user = User.objects.create(email="user@staff.com")
+        self.session_1 = Session.objects.create(season=Session.SPRING, year=2021)
+        self.session_2 = Session.objects.create(season=Session.SUMMER, year=2021)
```

Comment:
nit: can we rename to `session1` and `session2`?

Reply:
okie doke